pkg/resource/notebook_instance/testdata/test_suite.yaml

```diff
@@ -0,0 +1,358 @@
+         invoke: ReadOne
+         expect:
+           latest_state: "notebook_instance/v1alpha1/output/readone/nb_readone_inservice.yaml"
+       - name: "ReadOne=StoppedAfterUpdateACK"
+         description: "Testing how readone handles the Stopped after it's been updated. The Notebook here has been stopped by ack."
+         given: 
+           desired_state: "notebook_instance/v1alpha1/input/readone/nb_readone_stopped_after_update_ack.yaml"
+           svc_api:
+             - operation: DescribeNotebookInstanceWithContext
+               output_fixture: "notebook_instance/readOne/readone_stopped_after_update_ack.json"
+             - operation: StartNotebookInstance
+         invoke: ReadOne
+         expect:
+           latest_state: "notebook_instance/v1alpha1/output/readone/nb_readone_stopped_after_update.yaml"
+       - name: "ReadOne=StoppedAfterUpdateHuman"
+         description: "Testing how readone handles the Stopped state when stopped by human for an update, StoppedByControllerMeta should 
+         not be there."
+         given: 
+           desired_state: "notebook_instance/v1alpha1/input/readone/nb_readone_stopped_after_update_human.yaml"
+           svc_api:
+             - operation: DescribeNotebookInstanceWithContext
+               output_fixture: "notebook_instance/readOne/readone_stopped_after_update_ack.json"
+         invoke: ReadOne
+         expect:
+           latest_state: "notebook_instance/v1alpha1/output/readone/nb_readone_stopped_after_update.yaml"
+       - name: "ReadOne=StoppedBeforeUpdateACK"
+         description: "Testing how readone handles the Stopped state when it's stopped right before an update. StoppedByControllerMeta should be in 
+         UpdatePending."
+         given: 
+           desired_state: "notebook_instance/v1alpha1/input/readone/nb_readone_stopped_before_update_ack.yaml"
+           svc_api:
+             - operation: DescribeNotebookInstanceWithContext
+               output_fixture: "notebook_instance/readOne/readone_stopped_before_update_ack.json"
+         invoke: ReadOne
+         expect:
+           latest_state: "notebook_instance/v1alpha1/output/readone/nb_readone_stopped_before_update_ack.yaml"
+       - name: "ReadOne=StoppingDeleting"
+         description: "Testing how readone handles the Stopping stopping state before going into deleting"
+         given: 
+           desired_state: "notebook_instance/v1alpha1/input/readone/nb_readone_stopping_deleting.yaml"
+           svc_api:
+             - operation: DescribeNotebookInstanceWithContext
+               output_fixture: "notebook_instance/readOne/readone_stopping_deleting.json"
+         invoke: ReadOne
+         expect:
+           latest_state: "notebook_instance/v1alpha1/output/readone/nb_readone_stopping_deleting.yaml"
+       - name: "ReadOne=StoppedUpdate"
+         description: "Testing how readone handles the Stopping stopping state before going into Update"
```

Comment:
nit: Stopping stopping? repeated word or is this intendend

Reply:
It was a repeated word,  also fixed the test name